packages/spatial/src/physics/components/ColliderComponent.tsx

```diff
@@ -101,7 +101,7 @@ export const ColliderComponent = defineComponent({
         Physics.removeCollider(physicsWorld, entity)
         hasCollider.set(false)
       }
-    }, [physicsWorld, component.shape, rigidbodyEntity, !!rigidbodyComponent, transform.scale])
+    }, [physicsWorld, component.shape, rigidbodyEntity, !!physicsWorldRigidbody, transform.scale])
```

Comment:
!!physicsWorldRigidbody.value

Reply:
This doesnt return the wrapped state.

Reply:
nvm, useWorld() doesn't return a state-wrapped physics world..